docs/integrations/aws/index.md

```diff
@@ -32,8 +32,92 @@ In additional to runtime libraries installed by the Serverless Extension, an ext
 Lambda Extension is used to forward logs, metrics, and traces to
 Serverless Console. This component is independent from the language runtime.
 
-### Configuring the Serverless Console Extension
+## Configuring the Serverless Console Extension
 [Serverless Framework](../index.md) is the easiest way to instrument your app or
 service with the Serverless Console Extension. If your not using 
 Serverless Framework you'll need to build, and deploy the extension 
 yourself for each of your Lambdas functions. 
+
+### Pre-requisites
+Before you get started make sure you have the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+and a recent version of [Node.js](https://nodejs.dev/learn/how-to-install-nodejs) and a 
+[Serverless Console Organization](https://console.serverless.com?ref_website=https%3A%2F%2Fwww.serverless.com%2Fconsole%2Fdocs%2F) and [jq]().
+
+### 1.Clone the repo and install Node modules
+Once you have everything you need you'll need to clone this repo
+and install node modules.
+
+```text
+git clone https://github.com/serverless/console.git
+cd console
+npm install
+```
+
+### 2.Build the extension locally
+Next you'll need to run a set of build commands to build
+the internal and external extensions.
+
+```
+cd node/packages/aws-lambda-otel-extension/external/otel-extension-external
+npm install
+cd ../../internal/otel-extension-internal-node
+npm install
+cd ../../
+npm run build
+```
+This will generate an artifact in `dist/extension.zip`
+
+### 3.Publish the extension to your AWS Account
+Once you have built the extension, you can publish it 
+to your AWS account.
+
+```text
+aws lambda publish-layer-version \
+   --layer-name sls-otel-extension \
+   --zip-file fileb://./dist/extension.zip 
+   --region us-east-1 
+```
+
+### 4.Request an ingest token
+Next you'll need to request an ingest token based on your 
+organization id, service name, and environment you wish to
+deploy to. It's helpful to set these as environment variables.
+
+* Go to your org settings page to get your org id and org token
+* Set the following environment variables (used later)
+
+```text
+ORG_ID=<your-org-id> 
+ORG_TOKEN=<your-org-token>
+NAMESPACE=<your-namespace>
+FUNCTION_NAME<name-of-lambda-function>
+ENVIRONMENT=dev
+REGION=us-east-1
+```
+
+*  Curl the following endpoint to get an ingest token.
+```
+INGEST_TOKEN=$(curl -X POST \
+  'https://core.serverless-dev.com/ingestion/kinesis/token' \
+  --header 'Accept: application/json' \
+  --header 'sls-token-type: orgToken' \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: Bearer $ORG_TOKEN' \
+  --data-raw '{
+        "orgId": "$ORG_ID",
+        "serviceId": "test-service",
+        "stage": "$ENVIRONMENT"
+}' | jq -r .token.accessToken)
+```
```

Comment:
We need to add some details about the ingest token lifecycle to prevent customers from accidentally revoking a token they are currently using in another service or even on the same service once they decided they want to redeploy.

Here are a few rules/thing to keep in mind with these tokens

These tokens are rotated every 30 days if you make this same request with the same org, service, and stage to this endpoint.

For example 👇 

If you call this endpoint for the first time you will get a new token
If you call this endpoint some time after the first call within 30 days (Say on day 20) you will receive the same ingest token
If you call this endpoint some time after the first call and it has been more than 30 days (Say on day 35) you will receive a new ingest token and the old token will no longer be valid

Basically a good rule of thumb for these tokens is to generate 1 token per stack (Cloudformation, terraform, etc).

This is so you don't have to worry about deploying multiple stacks if you end up generating a new token with the 30 day policy I mentioned above.

Also it might be a good idea to not include this find/create token endpoint in any CI/CD process since a failed deploy might result in the existing service's token being revoked and not replaced if you call this endpoint outside of the 30 day period and the deployment fails.

Does this make sense?

Reply:
I think this makes sense and I'll add some additional context here. Thanks for the rundown.